swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -79,6 +79,7 @@ public final class Store: ObservableObject {
         // Try to load existing configuration
         if let manager = try await VPNConfigurationManager.load() {
           self.vpnConfigurationManager = manager
+          self.settings = try manager.asSettings()
```

Comment:
Not sure about Swift but in Rust `as` denotes a cheap conversion between references, no allocations etc.

Perhaps this is just a "settings" accessor?